controls/radmaskedinput/features/maskedinput-controls/currency.md

```diff
@@ -48,25 +50,111 @@ Here is a simple definition of a __RadMaskedCurrencyInput__ control:
 ![](images/radmaskedinput_currencyinput_default.png)
 
 You can further customize the __RadMaskedCurrencyInput__ control's behavior by setting the IsCurrencySymbolVisible property to False thus hiding the culture specific currency symbol. By default this property is set to True.
-				
-You can further restrict the user input by setting the __FormatString__ property.It uses [Standard Numeric Format Strings](http://msdn.microsoft.com/en-us/library/dwhawy9k.aspx) and [Custom Numeric Format Strings](http://msdn.microsoft.com/en-us/library/0c899ak8.aspx) to further format the __Text__ property.				
 
-#### __[XAML] Example 2: Setting the FormatString property__
-{{region xaml-radmaskedinput-features-controls-currency_1}}
+## Properties
+
+* __Value__: A property of type __decimal?__ that gets or sets the current value of the control.
+* __AutoFillNumberGroupSeparators__: A boolean property that gets or sets whether number group separators are auto filled in the mask.
+* __AutoFillZeros__: A boolean property that gets or sets whether trailing zeros should be auto filled.
+* __AllowSkipPlaceholders__: A boolean property that gets or sets whether input can skip placeholders.
+* __IsCurrencySymbolVisible__: A boolean property that gets or sets whether the currency symbol is visible or not. Default value is __True__.
+
+## Data Binding
+
+RadMaskedCurrencyInput's __Value__ property is of type __nullable decimal (decimal?)__ and you have to bind it to ViewModel's property of type decimal or nullable decimal (if you need to set null). 
+
+>important Binding to __object__ is not support and may result in unpredictable behavior.
+
+#### __[XAML] Example 2: Define the view model__
+{{region radmaskedinput-features-controls-currency_1}}
+	public class ViewModel : ViewModelBase
+	{
+		private decimal? amount;
+		
+		public ViewModel()
+		{
+			this.amount = 12345.67;
+		}
+		
+		public decimal? Amount
+        {
+            get { return this.amount; }
+            set
+            {
+                if(this.amount !=  value)
+                {
+                    this.amount = value;
+                    this.OnPropertyChanged("Amount");
+                }           
+            }
+        }	
+	}
+{{endregion}}
+
+#### __[C#] Example 3: Binding the Value property__
+{{region radmaskedinput-features-controls-currency_2}}
+	<telerik:RadMaskedCurrencyInput Culture="en-US"
+								   InputBehavior="Replace"
+								   Mask="#9.2"
+								   TextMode="PlainText"
+								   UpdateValueEvent="LostFocus"
+								   Value="{Binding Amount,Mode=TwoWay}" />
+{{endregion}}
+
+
+## FormatString property
+
+You can further restrict the user input by setting the __FormatString__ property.It uses [Standard Numeric Format Strings](http://msdn.microsoft.com/en-us/library/dwhawy9k.aspx) and [Custom Numeric 

```

Comment:
I don't think the FormatString __restricts__ the input. Can you double check this?